### PR TITLE
Alt text audit 

### LINKED ISSUE #1192: Confirm every image has an alt attribute
### Overview
We need an `alt` attribute for accessibility purposes for each image. Some images throughout the site do not have `alt` attributes

### Action Items
- [x] Confirm every image has an `alt` attribute
- [x] Add appropriate descriptions for images without `alt` attributes

### Resources/Instructions
https://www.w3schools.com/tags/att_img_alt.asp

### PR DESCRIPTION
**Pull request by qiqicodes**
fixed #1192 

Action Items
- [ ] Confirm every image has an alt attribute
- [ ] Add appropriate descriptions for images

**Why is alt text important?**
* Alt texts take place of the image, when user choose to lower their data usage. 
* Screen readers read alt texts for users who may or may not have access to the images available.
 
**Resources**
* [WCAG 2.0 Guideline](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#text-alternatives)

**Tips**
- [ ] For informational non-text content, as you review, try to imagine the image presented if the alt text can recreate the image.
- [ ] For decorative image, keep alt="" (i.e. blue background is negligible).
- [ ] For more details, refer to WCAG 2.0 guideline.

### CODE REVIEW

File: _projects/record-clearance-project.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ title: Record Clearance Project
 description: 'The Record Clearance Project aims to help people in California with criminal records accomplish record clearance, expungement or reduction and subsequently a second chance as a part of society.
 <br /><br />We work with verifiable non-profits, the government and partners to build digital tools that can affect changes in the lives of these justice impacted individuals.'
 image: /assets/images/projects/record-clearance.jpg
-alt: 'Record clearance project'
+alt: 'Record clearance project logo'
```
Comment (jbubar) [inline _projects/record-clearance-project.md:7]: just thought I would point this one out.. lol
Just popped out in contrast to all you other changes. Made me laugh

**Review by jbubar — COMMENTED** [review]
Pretty extensive pull request!
Great work.. just finished my first pass through. 
I read all the changes, and they are all really well written and descriptive.
I will test out all the pages tomorrow.

File: pages/about.html [hunk 2]
```diff
@@ -192,25 +192,25 @@ <h1>About Us</h1>
...
+                                <img src="/assets/images/about/sdg-elements/zero-hunger.svg" alt="Zero Hunger goal logo" />
+                                <img src="/assets/images/about/sdg-elements/gender-equality.svg" alt="Gender Equality goal logo" />
+                                <img src="/assets/images/about/sdg-elements/decent-work.svg" alt="Decent Work and Economic Growth goal logo" />
+                                <img src="/assets/images/about/sdg-elements/sustainable-cities.svg" alt="Sustainable Cities and Communities goal logo" />
+                                <img src="/assets/images/about/sdg-elements/climate-action.svg" alt="Climate Action goal logo" />
+                                <img src="/assets/images/about/sdg-elements/peace-justice.svg" alt="Peace, Justice and Strong Intuitions goal logo" />
+                                <img src="/assets/images/about/sdg-elements/partnerships.svg" alt="Partnerships for the goals goal logo" />
+                                <div class="grid-center"><img src="assets/images/about/sdg-elements/sdg-la.svg" alt="Los Angeles Sustainable Development wheel logo" /><br />
```
Comment (ItsLaszlo) [inline pages/about.html]: missing the beginning **/** in the src field

`src="/assets/images/about/sdg-elements/sdg-la.svg"`

Screen Shot 2021-04-05 at 8 47 56 PM

File: pages/about.html [hunk 3]
```diff
@@ -192,25 +192,25 @@ <h1>About Us</h1>
...
                                     <a href="https://sdg.lamayor.org"><span class="small-text">https://sdg.lamayor.org</span></a>
                                 </div>
                             </div>
                         </div>
                     </div>
-                </div>
+                </div>/
```
Comment (ItsLaszlo) [inline pages/about.html]: need to remove the **/**

**Review by ItsLaszlo — CHANGES_REQUESTED** [review]

**Comment by akibrhast** [comment]
@qiqicodes Could you please resolve the merge conflicts and apply the changes as requested by @martham0

File: _projects/record-clearance-project.md (hunk 1 above)
Comment (qiqicodes) [reply _projects/record-clearance-project.md:7]: sometimes, logo is all we need xD

File: pages/about.html (hunk 2 above)
Comment (qiqicodes) [reply pages/about.html]: Thanks Martha, nice catch!

**Comment by jbubar** [comment]
@qiqicodes could you resolve the merge conflict again?

also, this happened when I pulled it so jekyll did not render the files 
Screen Shot 2021-04-13 at 11 36 23 AM
not sure why.. I looked at the files and nothing seemed off.  could you run it locally on your machine and see if you get any similar errors?

**Review by jbubar — APPROVED** [review]
Nice! Ready to merge, I looked through all the projects, and pages... I hope.. This was a lot to review

**Pull request merged**